tests/scripts/thread-cert/Cert_9_2_07_DelayTimer.py

```diff
@@ -43,7 +43,7 @@
 ROUTER_PENDING_TIMESTAMP = 30
 ROUTER_PENDING_ACTIVE_TIMESTAMP = 21
```

Comment:
When the Pending Dataset Delay Timer expires, its contents are applied to the Active Dataset, including the Active Timestamp itself.